provider/cmd/pulumi-resource-aws-native/semantics.json

```diff
@@ -0,0 +1,22 @@
+{
+    "resources": {
+        "aws-native:sqs:Queue": {
+            "cf": "AWS::SQS::Queue",
+            "namingTriviaSpec": 
+            {
+                "queueName": {
+                    "rule": {
+                        "field": "fifoQueue",
+                        "condition": {
+                            "predicate": "equal",
+                            "value": { "V": true }
```

Comment:
I went back and forth a bit wrt whether this ought to be an explicit `PropertyValue` in the spec, or if we ought to coerce it into one as we parse the spec.  I'm sort of indifferent, but if there are any strong opinions against this pattern, I'm happy to change it.